### PR TITLE
Add a way to get the timescale of an avis

### PR DESCRIPTION
**Pull request by padenot**
I missed this in the previous PR.

Somehow, github doesn't want to mark the file as renamed even in a separate commit.

### CODE REVIEW

File: mp4parse_capi/tests/test_avis.rs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+fn no_edts() {
+    check_loop_count("tests/no_edts.avif", -1);
+}
+
+#[test]
+fn check_timescales() {
+    check_timescale("tests/loop_1.avif", 2);
+    check_timescale("tests/loop_2.avif", 2);
+    check_timescale("tests/loop_ceiled_4.avif", 2);
+    check_timescale("tests/loop_forever.avif", 2);
```
Comment (padenot) [inline mp4parse_capi/tests/test_avis.rs:111]: Weird timescales, but correct when verified with https://gpac.github.io/mp4box.js/test/filereader.html.

File: mp4parse_capi/tests/test_avis_loop_count.rs [hunk 2]
```diff
@@ -85,3 +102,12 @@ fn loop_forever() {
 fn no_edts() {
     check_loop_count("tests/no_edts.avif", -1);
 }
+
+#[test]
+fn check_timescales() {
+    check_timescale("tests/loop_1.avif", 2);
+    check_timescale("tests/loop_2.avif", 2);
+    check_timescale("tests/loop_ceiled_4.avif", 2);
+    check_timescale("tests/loop_forever.avif", 2);
```
Comment (padenot) [inline mp4parse_capi/tests/test_avis_loop_count.rs]: Weird but correct timescale. I haven't been able to figure out yet what tool is able to create an `avis` file that has a top-level timescale and a track timescale, for testing, `ffmpeg` doesn't.

File: mp4parse_capi/src/lib.rs [hunk 3]
```diff
@@ -1297,6 +1297,7 @@ pub unsafe extern "C" fn mp4parse_avif_get_indice_table(
     parser: *mut Mp4parseAvifParser,
     track_id: u32,
     indices: *mut Mp4parseByteData,
+    timescale: *mut u64,
```
Comment (kinetiknz) [inline mp4parse_capi/src/lib.rs:1300]: Add an `is_null()` check for this please.

**Review by kinetiknz — APPROVED** [review]
LGTM, thanks! One small change needed before merging.

**Pull request merged**